table/block_based/block_based_table_reader_test.cc

```diff
@@ -36,12 +36,21 @@ class BlockBasedTableReaderBaseTest : public testing::Test {
   // If mixed_with_human_readable_string_value == true,
   // then adjacent blocks contain values with different compression
   // complexity: human readable strings are easier to compress than random
-  // strings.
-  static std::map<std::string, std::string> GenerateKVMap(
-      int num_block = 100, bool mixed_with_human_readable_string_value = false,
-      size_t ts_sz = 0) {
-    std::map<std::string, std::string> kv;
-
+  // strings. key is an internal key.
+  // When ts_sz > 0 and `same_key_diff_ts` is true, this
+  // function generate keys with the same user provided key, with different
+  // user defined timestamps and different sequence number to differentiate them
+  static std::vector<std::tuple<std::string, std::string>> GenerateKVMap(
```

Comment:
Minor but `std::pair<std::string, std::string>` would also work here and would eliminate the need to switch to `std::get` in several places

Reply:
Good call! That simplifies the syntax.